docs/flow-exact-props.md

```diff
@@ -1,41 +1,35 @@
-# Require exact object types for Props (flow-exact-props)
+# Require exact object types for State (flow-exact-state)
```

Comment:
I think you flipped the contents of these two files!

Reply:
Lol, I copy pasted and modified the wrong one. 😅 
Thanks for catching this.